### DISCUSSION TITLE
4 Semester 2 Woche

### DISCUSSION THREAD

**Discussion by MelonigeMelone** [MelonigeMelone/xCalendar#27] [Blogkanal]
# RMMM-Liste

Inder RMMM-Liste finden sich zwei technische Risiken, welche eventuell bei unserer Kalendar App auftreten könnten.

<img width="1328" alt="image" src="https://github.com/MelonigeMelone/xCalendar/assets/52330056/725788aa-f92b-4cc3-acb9-39c669c1c27b">

**Comment by Bjoern04** [comment]
Das sieht schon sehr gut aus. Die beiden Risiken sind einleuchtend. 
Bei den Verantwortlichen soll glaube ich aber eine einzelne Person eingetragen werden und nicht ein ganzes Team.

**Comment by Christian-2003** [comment]
Hallo zusammen,
dem was @Bjoern04 bereits gesagt hat, kann ich mich nur anschließen.
Allerdings erscheint mit die Klassifizierung der Wahrscheinlichkeit des Verlustes von Benutzerdaten als "Mittel" recht besorgniserregend... :sweat_smile: Hier würde es sich vielleicht lohnen daran zu arbeiten, sodass die Wahrscheinlichkeit des Risikos sinkt.

**Comment by YPuest** [comment]
Moin,
kann mich @Christian-2003 und @Bjoern04 nur anschließen. Mich würde aber interessieren, ob dass die einzigen Risiken in eurem Projekt sind?
